readme_template.md

```diff
@@ -1,6 +1,8 @@
 ----
-**Take Note!**  This version of the Hosts file generator, and tests, are for Python 3.5+ only.
-
+**Take Note!**
+- This version of the Hosts file generator, and tests, are for Python 3.5+ only.
+- This repository centers around the hosts file aggregation Python script and NOT around the hosts files it produces. Issues regarding the personal management of hosts files, such as hosts file managers, are out of scope. However, as a courtesy, there are some recommended hosts manager options available.
+- With the exception of issues and PRs regarding changes to `hosts/data/StevenBlack/hosts`, all other issues regarding the content of the produced hosts files should be made with the appropriate data source that contributed the content in question. The contact information for all of the data sources can be found in the `hosts/data/` directory.
```

Comment:
I think this is basically good, but I doubt this will help.  Because this supposes pre-existing knowledge of the repo's folder structure, and its machinations, both of which are unconventional and not obvious from the repo home page.